packages/tslint-config/index.js

```diff
@@ -25,11 +25,6 @@ module.exports = {
                 // ["assert", "equal", "use assert.strictEqual instead"]
```

Comment:
this file should extend `blueprint-rules.js`, right?

Reply:
no i thought that was the whole point of this... that the base config does not include any blueprint-specific stuff?

Reply:
nah, the purpose is the inverse of that actually, that you can use the blueprint-specific rules (such as no icon strings) without pulling in the rest of the config

Reply:
ok so default config _includes_ blueprint, but you can also just use blueprint without config?

Reply:
yep! because most apps using BP are going to want to pick up the BP rules and config that say "don't use the .pt- prefix" but not pick up the general TS lint rule config